llm_mining_core/config/server.py

```diff
@@ -6,8 +6,9 @@
 from openai import OpenAI
 
 class LLMServerConfig:
-    MAX_MODEL_LEN = 8192
-    CHAT_TEMPLATE = "{% for message in messages %}{{'<|im_start|>' + message['role'] + '\n' + message['content'] + '<|im_end|>' + '\n'}}{% endfor %}{% if add_generation_prompt %}{{ '<|im_start|>assistant\n' }}{% endif %}"
+    MAX_MODEL_LEN = 4096
+    # TODO: we need to set the chat template??
```

Comment:
don't think we need this chat template. and let's use 8192 context length

Reply:
delete CHAT_TEMPLATE and update "MAX_MODEL_LEN = 8192"